### ISSUE TITLE
fcrondyn cutting long lines

### ISSUE THREAD

**Issue by Massimo-B** [yo8192/fcron#23]
```
fcrondyn -V
fcrondyn 3.3.1 - interact dynamically with daemon fcron
```

fcrondyn is cutting long lines:

```
# fcrontab -u root -l
2023-11-15 08:09:30  INFO listing root's fcrontab

@first(20),mail(1)      1w              smartctl -a /dev/disk/by-id/ata-Samsung_SSD_860_EVO_1TB_S3Z9NY0M457614P |grep Wear |tee ~/"wearlevelcount.$(date  -I)"
@mail(0)                1h              /usr/local/bin/1update_btrbksnapshotlinks /mnt/archive/*/* /mnt/local/data/archive/*/*

!mail(false),lavgand,lavg(12,10,8),until(2h),nice(15),first(5m)
%daily,first(45m)       * 05-10,14-17   ionice -c 3 schedtool -D -e btrbk -c /etc/btrbk/btrbk.conf run cron && /usr/local/bin/1update_btrbksnapshotlinks -c /etc/btrbk/btrbk.conf /mnt/archive/*/* /mnt/local/data/archive/*/*
```

```
# fcrondyn -x ls
ID   |USER     |SCHEDULE        |CMD
0    |systab   |2023-11-15 09:00|run-parts /etc/cron.hourly
16   |root     |2023-11-15 09:12|/usr/local/bin/1update_btrbksnapshotlinks /mnt/archive/*/* /mnt/local/data/archive/*/*
1    |systab   |2023-11-16 00:00|run-parts /etc/cron.daily
4    |clamav   |2023-11-16 05:00|fangfrisch -c /etc/fangfrisch.conf refresh
15   |root     |2023-11-16 05:00|ionice -c 3 schedtool -D -e btrbk -c /etc/btrbk/btrbk.conf run cron && /usr/local/bin/1update_btrbksnapshotlinks -c /etc/btrbk/btrbk.conf /mnt/archive/*/* /mnt/lo17   |root     |2023-11-17 05:01|smartctl -a /dev/disk/by-id/ata-Samsung_SSD_860_EVO_1TB_S3Z9NY0M457614P |grep Wear |tee ~/"wearlevelcount.$(date  -I)"
2    |systab   |2023-11-20 00:00|run-parts /etc/cron.weekly
3    |systab   |2023-12-01 00:00|run-parts /etc/cron.monthly
```

**Comment by yo8192** [comment]
Thanks for pointing that out. Indeed, that needs fixing, I'll have a look.

**Comment by yo8192** [comment]
Now fixed with commit c92e985f827daf34d0a5bdcef9d50f027c7c6004.
